src/studio/src/designer/backend/Services/Implementation/GiteaAPIWrapper.cs

```diff
@@ -67,6 +72,25 @@ public async Task<User> GetCurrentUser()
             return null;
         }
 
+        /// <inheritdoc />
+        public async Task<List<Team>> GetTeams()
+        {
+            List<Team> teams = new List<Team>();
+
+            Claim claim = _httpContextAccessor.HttpContext.User.Claims.FirstOrDefault(c => c.Type == AltinnCoreClaimTypes.DeveloperToken);
+
+            if (claim == null)
+            {
+                return teams;
+            }
+
+            string url = $"user/teams?token={claim.Value}";
```

Comment:
Is this a separate API from other operations? The _httpClient already have the token as an Authorization header.

Reply:
Hmm, never tested without the token. Just copied the code I used towards the API in postman. Will test without